pymc3/tests/test_distributions.py

```diff
@@ -1684,10 +1684,15 @@ def test_lkj(self, x, eta, n, lp):
         decimals = select_by_precision(float64=6, float32=4)
         assert_almost_equal(model.fastlogp(pt), lp, decimal=decimals, err_msg=str(pt))
 
-    @pytest.mark.parametrize("n", [2, 3])
+    @pytest.mark.parametrize("n", [1, 2, 3])
```

Comment:
I also added an extra test to check for last dimensional shape being (1, ) and to match the logp from scipy.